main_loop_tf/validate.py

```diff
@@ -121,14 +145,20 @@ def validate(placeholders,
             # Get batch pred, mIoU so far, batch loss and potentially the
             # summary
             if cidx % cfg.val_summary_freq == 0:
+                v_outs = eval_outs[0] + [val_summary_op[0]]
```

Comment:
Actually if you do it before the set_has_GT if you can then reuse it also in the non set_has_GT branch at L 177